assets/apps/components/src/Controls/FormTokenFieldControl.js

```diff
@@ -8,16 +12,29 @@ const FormTokenFieldControl = ({
 	onChange,
 	value,
 }) => {
+	const REQUEST_DEBOUNCE_DELAY = 400;
+	const MAX_TERMS_SUGGESTIONS = 20;
+	const DEFAULT_QUERY = {
+		per_page: MAX_TERMS_SUGGESTIONS,
+		order: 'asc',
+		_fields: 'id,slug',
+		context: 'view',
+	};
```

Comment:
I agree with @mskapusuz. The suggestions should use the post title, not the slug.

Reply:
The FormTokenField accepts as suggestions only a simple array ( eg. an array of ids, an array of slugs, an array of titles, etc. ).
For the suggestions, I must pass to the control the labels that will appear on the front end. When a user selects a new value, I must correlate the selected label with an ID. Since the title is not unique ( two posts can have the same title), I choose the slug instead. So for REST, I would actually need "slugs that start with".

Reply:
I still think it's bad UX to show the slug instead of the title. Even if titles can be the same, the probability of that happening is really low.